bin/security/setupSecurityAccount.sh

```diff
@@ -2,20 +2,41 @@
 
 option="${1}"
 
+# x86_64 or arm64 for make run option
+[ "$(uname -m)" != "x86_64" ] && USE_ARM64="-arm64"
+
 BB_ROOT_DIR=$(cd $(dirname "$0"); cd ..; pwd)
-COMPOSE=$BB_ROOT_DIR/$(ls $BB_ROOT_DIR | awk '/docker-compose/ && !/test-tools/')
+MAKEFILE=$BB_ROOT_DIR/Makefile
 
 case ${option} in
   -useradd)
   echo "Info: Add security account."
-  OT=$(docker-compose -f ${COMPOSE} run --rm \
-    --entrypoint /edgex/security-proxy-setup edgex-proxy --init=false --useradd=geneva --group=admin | grep '^the access token for')
-  export TOKEN=$(echo ${OT} | sed 's/.*: \([^.]*\.[^.]*\.[^.]*\).*/\1/')
+  TOKEN=`docker run --rm \
+		-e SECRETSERVICE_SERVER=edgex-vault \
+		-e KONGURL_SERVER=kong \
+		-e SECRETSERVICE_TOKENPATH=/tmp/edgex/secrets/edgex-security-proxy-setup/secrets-token.json \
+		-e SECRETSERVICE_CACERTPATH=/tmp/edgex/secrets/ca/ca.pem \
+		--network edgex_edgex-network \
+		--volume /tmp/edgex/secrets/ca:/tmp/edgex/secrets/ca:ro,z \
+		--volume /tmp/edgex/secrets/edgex-security-proxy-setup:/tmp/edgex/secrets/edgex-security-proxy-setup:ro,z \
+		nexus3.edgexfoundry.org:10004/docker-edgex-security-proxy-setup-go${USE_ARM64}:master \
+		--init=false --useradd=testinguser --group=admin \
+		| grep "access token for user testinguser is" | sed 's/.*: \([^.]*\.[^.]*\.[^.]*\).*/\1/'`
+  echo TOKEN=$TOKEN
```

Comment:
Since we get the entire docker-compose.yml from the developer-scripts repo, could we just use `docker-compose -p edgex` to access token?

Reply:
> could we just use `docker-compose -p edgex` to access token?

This is more efficient, but does require more knowledge of internals of compose file... So yes... ;-)

Reply:
Done.